.nyc_output/3feadddc-a951-4e57-a7b8-4217232779ca.json

```diff
@@ -0,0 +1 @@
{"line":215,"column":16},"end":{"line":215,"column":82}},"112":{"start":{"line":216,"column":16},"end":{"line":216,"column":52}},"113":{"start":{"line":218,"column":16},"end":{"line":218,"column":23}},"114":{"start":{"line":222,"column":8},"end":{"line":236,"column":11}},"115":{"start":{"line":223,"column":30},"end":{"line":223,"column":32}},"116":{"start":{"line":225,"column":26},"end":{"line":227,"column":45}},"117":{"start":{"line":229,"column":12},"end":{"line":235,"column":15}},"118":{"start":{"line":230,"column":16},"end":{"line":230,"column":60}},"119":{"start":{"line":231,"column":16},"end":{"line":231,"column":72}},"120":{"start":{"line":232,"column":16},"end":{"line":232,"column":52}},"121":{"start":{"line":234,"column":16},"end":{"line":234,"column":23}}},"fnMap":{"0":{"name":"(anonymous_0)","decl":{"start":{"line":5,"column":23},"end":{"line":5,"column":24}},"loc":{"start":{"line":5,"column":29},"end":{"line":238,"column":1}},"line":5},"1":{"name":"(anonymous_1)","decl":{"start":{"line":24,"column":28},"end":{"line":24,"column":29}},"loc":{"start":{"line":24,"column":34},"end":{"line":29,"column":5}},"line":24},"2":{"name":"(anonymous_2)","decl":{"start":{"line":25,"column":53},"end":{"line":25,"column":54}},"loc":{"start":{"line":25,"column":59},"end":{"line":28,"column":9}},"line":25},"3":{"name":"(anonymous_3)","decl":{"start":{"line":31,"column":27},"end":{"line":31,"column":28}},"loc":{"start":{"line":31,"column":33},"end":{"line":37,"column":5}},"line":31},"4":{"name":"(anonymous_4)","decl":{"start":{"line":32,"column":42},"end":{"line":32,"column":43}},"loc":{"start":{"line":32,"column":48},"end":{"line":36,"column":9}},"line":32},"5":{"name":"(anonymous_5)","decl":{"start":{"line":39,"column":26},"end":{"line":39,"column":27}},"loc":{"start":{"line":39,"column":32},"end":{"line":86,"column":5}},"line":39},"6":{"name":"(anonymous_6)","decl":{"start":{"line":40,"column":48},"end":{"line":40,"column":49}},"loc":{"start":{"line":40,"column":58},"end":{"line":57,"column":9}},"line":40},"7":{"name":"(anonymous_7)","decl":{"start":{"line":51,"column":57},"end":{"line":51,"column":58}},"loc":{"start":{"line":51,"column":69},"end":{"line":56,"column":13}},"line":51},"8":{"name":"(anonymous_8)","decl":{"start":{"line":59,"column":37},"end":{"line":59,"column":38}},"loc":{"start":{"line":59,"column":47},"end":{"line":71,"column":9}},"line":59},"9":{"name":"(anonymous_9)","decl":{"start":{"line":64,"column":58},"end":{"line":64,"column":59}},"loc":{"start":{"line":64,"column":69},"end":{"line":70,"column":13}},"line":64},"10":{"name":"(anonymous_10)","decl":{"start":{"line":73,"column":33},"end":{"line":73,"column":34}},"loc":{"start":{"line":73,"column":43},"end":{"line":85,"column":9}},"line":73},"11":{"name":"(anonymous_11)","decl":{"start":{"line":78,"column":58},"end":{"line":78,"column":59}},"loc":{"start":{"line":78,"column":69},"end":{"line":84,"column":13}},"line":78},"12":{"name":"(anonymous_12)","decl":{"start":{"line":88,"colu
```

Comment:
Remove this file and add ` .nyc_output/` to gitignore

Reply:
Done